Detectors/TPC/base/include/TPCBase/CDBInterface.h

```diff
@@ -138,9 +141,11 @@ const std::unordered_map<CDBType, const std::string> CDBTypeMap{
   // ITPCCs
   {CDBType::CalITPC0, "TPC/Calib/ITPCC_0"},
   {CDBType::CalITPC1, "TPC/Calib/ITPCC_1"},
-  // correction maps
+  // correction maps for static distortions
```

Comment:
Static distortions are only the `Ref` maps
```suggestion
  // correction maps
```